authentication/managing_cloud_provider_credentials/cco-mode-sts.adoc

```diff
@@ -0,0 +1,734 @@
+                    "ec2:DescribeAvailabilityZones",
+                    "ec2:DescribeDhcpOptions",
+                    "ec2:DescribeImages",
+                    "ec2:DescribeInstances",
+                    "ec2:DescribeSecurityGroups",
+                           "ec2:DescribeSubnets",
+        "ec2:DescribeVpcs",
+                    "ec2:RunInstances",
+                    "ec2:TerminateInstances",
+                    "elasticloadbalancing:DescribeLoadBalancers",
+                    "elasticloadbalancing:DescribeTargetGroups",
+                    "elasticloadbalancing:RegisterInstancesWithLoadBalancer",
+                    "elasticloadbalancing:RegisterTargets",
+                    "iam:PassRole",
+                    "iam:CreateServiceLinkedRole"
+                ],
+                "Resource": "*"
+            },
+            {
+                "Effect": "Allow",
+                "Action": [
+                    "kms:Decrypt",
+                    "kms:Encrypt",
+                    "kms:GenerateDataKey",
+                    "kms:GenerateDataKeyWithoutPlainText",
+                    "kms:DescribeKey"
+                ],
+                "Resource": "*"
+            },
+            {
+                "Effect": "Allow",
+                "Action": [
+                    "kms:RevokeGrant",
+                    "kms:CreateGrant",
+                    "kms:ListGrants"
+                ],
+                "Resource": "*",
+                "Condition": {
+                    "Bool": {
+                        "kms:GrantIsForAWSResource": true
+                    }
+                }
+            }
+        ]
+    }
+}
+----
+. Prepare to run the {product-title} installer:
+
+.. Create the `install-config.yaml` file:
++
+[source,terminal]
+----
+$ ./openshift-install create install-config
+----
+
+.. Configure the cluster to install with the CCO in manual mode:
++
+[source,terminal]
+----
+$ echo "credentialsMode: Manual" >> install-config.yaml
+----
+
+.. Create install manifests:
++
+[source,terminal]
+----
+$ ./openshift-install create manifests
+----
+
+.. Create a `tls` directory, and copy the private key generated previously there:
++
+[NOTE]
+====
+The target file name is important.
+====
++
+[source,terminal]
+----
+$ mkdir tls ; cp <path-to-service-account-signer> ./tls/bound-service-account-signing-key.key
+----
+
+.. Create a custom `Authentication.config.openshift.io` CR with the file name `cluster-authentication-02-config.yaml`:
++
+[source,terminal]
+----
+$ cat << EOF > manifests/cluster-authentication-02-config.yaml
+iVersion: config.openshift.io/v1
+kind: Authentication
+metadata:
+  name: cluster
+spec:
+  serviceAccountIssuer: $OPENID_BUCKET_URL
+EOF
+----
+
+.. For each `CredentialsRequest` CR that is extracted from the release image, create a secret with the target namespace and target name that is indicated in each `CredentialsRequest`, substituting the AWS IAM role ARN created previously for each component:
```

Comment:
Do we need to add the `oc adm release extract` command again as a reminder?

Reply:
It can't hurt!